src/cpp/include/openvino/genai/visual_language/pipeline.hpp

```diff
@@ -29,7 +31,23 @@ class OPENVINO_GENAI_EXPORTS VLMPipeline {
         const ov::AnyMap& properties = {}
     );
 
-    /// @brief Construct a pipeline form a folder containing tokenizer
+    /// @brief Construct a pipeline from a map of models and their weights.
+    /// @param models_map A map where key is model name (e.g. "vision_embeddings", "text_embeddings", "language", "resampler")
+    /// and value is a pair of model IR as string and weights as tensor.
+    /// @param tokenizer A tokenizer.
+    /// @param config_dir_path A path to directory containing config.json.
+    /// @param device Inference device. A tokenizer is always compiled
+    /// for CPU.
+    /// @param properties A config to pass to ov::Core::compile_model().
+    VLMPipeline(
+        const ModelsMap& models_map,
+        const Tokenizer& tokenizer,
+        const std::filesystem::path& config_dir_path,
+        const std::string& device,
+        const ov::AnyMap& properties = {}
```

Comment:
in similar PR for LLMPipeline, @pavel-esir has added optional GenerationConfig https://github.com/openvinotoolkit/openvino.genai/pull/1262, I think we need to be aligned here

Reply:
please, add templated version as well (one with `template <typename... Properties, typename std::enable_if<ov::util::StringAny<Properties...>::value, bool>::type = true>`  instead of `const ov::AnyMap& properties`)

Reply:
Fixed